src/components/__tests__/example.js

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import { render } from "@testing-library/react"
+
+// You have to write data-testid
+const Title = () => <h1 data-testid="hero-title">Gatsby is awesome!</h1>
+
+test("Displays the correct title", () => {
+  const { getByTestId } = render(<Title />)
+  // Assertion
+  expect(getByTestId("hero-title")).toHaveTextContent("Gatsby is awesome!")
+  // --> Test will pass
+})
```

Comment:
One minor, and definitely not showstopping, piece of feedback here is that tests for [Appearance & Disappearance](https://testing-library.com/docs/guide-disappearance) can be done using the `toBeInTheDocument` matcher from `jest-dom`.

```suggestion
const Title = () => <h1>Gatsby is awesome!</h1>

test("Displays the correct title", () => {
  const { getByText } = render(<Title />)
  // Assertion
  expect(getByText("Gatsby is awesome!")).toBeInTheDocument()
  // --> Test will pass
})
```